File: cases/taint/go/local-chain-positive/local_chain_positive.go
package dataflowbench

func dfb_source() int { // DFB-SOURCE: local-chain-input
	return 1
}

func dfb_sink(value int) {} // DFB-SINK: local-chain-sink

func run() {
	first := dfb_source()
	second := first // DFB-WITNESS: local-chain-second
	third := second // DFB-WITNESS: local-chain-third
	dfb_sink(third)
}

File: cases/taint/kotlin/local-chain-positive/LocalChainPositive.kt
package dataflowbench

object LocalChainPositive {
    fun dfb_source(): Int { // DFB-SOURCE: local-chain-input
        return 1
    }

    fun dfb_sink(value: Int) {} // DFB-SINK: local-chain-sink

    fun run() {
        val first = dfb_source()
        val second = first // DFB-WITNESS: local-chain-second
        val third = second // DFB-WITNESS: local-chain-third
        dfb_sink(third)
    }
}
